Update setting-up-playbooks.rst (#5473)
mattermost/docs: source/playbooks/setting-up-playbooks.rst

 * To access the playbook dashboard, select the hyperlinked playbook name.

4. Configure the playbook the same way a playbook is created or edited.

Copying and reusing playbooks
-----------------------------

Playbooks are repeatable workflows and sometimes it's easier to copy and improve rather than start from scratch. You can do this by duplicating a playbook in the **Playbooks** screen. Select the **...** under **Actions** and then select **Duplicate**. The copied playbook will have **Copy of** appended to its original name which you can edit.

> You can import playbooks between teams. From the **Playbooks** screen, select **Import**. Choose the team you're importing to, then select the JSON file. You can also export any playbook to JSON to easily share with other Mattermost servers.

Reviewer: Not sure what `between teams` means here. You can import playbooks.

> You can import playbooks between teams. From the **Playbooks** screen, select **Import**. Choose the team you're importing to, then select the JSON file. You can also export any playbook to JSON to easily share with other Mattermost servers.

Author: ```suggestion
To import a playbook, go to the **Playbooks** screen, select **Import**. Choose the team you're importing to, then select the JSON file. You can also export any playbook to JSON to easily share with other Mattermost servers.
```

 * To access the playbook dashboard, select the hyperlinked playbook name.

4. Configure the playbook the same way a playbook is created or edited.

Copying and reusing playbooks
-----------------------------

Playbooks are repeatable workflows and sometimes it's easier to copy and improve rather than start from scratch. You can do this by duplicating a playbook in the **Playbooks** screen. Select the **...** under **Actions** and then select **Duplicate**. The copied playbook will have **Copy of** appended to its original name which you can edit.

You can also import playbooks between teams. From the **Playbooks** screen, select **Import**. Choose the team you're importing to, then select the JSON file.

> Lastly, if you want to create a backup of your playbook in JSON format, you can export it. To do this from the Playbooks screen, select the **...** under **Actions** and then select **Export**. Save the file to an appropriate location. The exported file can be used with your source control tool, to manage changes made to the playbook and allow you to rollback to previous versions.

Author: Ah interesting. Tao mentioned a use case around that but I may very well have misunderstood. :)

 * To access the playbook dashboard, select the hyperlinked playbook name.

4. Configure the playbook the same way a playbook is created or edited.

Copying and reusing playbooks
-----------------------------

Playbooks are repeatable workflows and sometimes it's easier to copy and improve rather than start from scratch. You can do this by duplicating a playbook in the **Playbooks** screen. Select the **...** under **Actions** and then select **Duplicate**. The copied playbook will have **Copy of** appended to its original name which you can edit.

> You can also import playbooks between teams. From the **Playbooks** screen, select **Import**. Choose the team you're importing to, then select the JSON file.

Author: @crspeller, could you help with any missing content/context here. Thank you!
